tiny11 creator 22621.525.bat

```diff
@@ -16,8 +16,8 @@ if not exist "%DriveLetter%\sources\boot.wim" (
 	echo.Please enter the correct DVD Drive Letter..
 	goto :Stop
 )
-
-if not exist "%DriveLetter%\sources\install.wim" (
+REM Check for install.esd instead of install.wim
+if not exist "%DriveLetter%\sources\install.esd" (

```

Comment:
A question is, how to processing if a windows install iso only has "install.wim"(iso dumpped and converted from uup) or "install.esd" (iso downloaded from microsoft)?

Reply:
You will have to look for either install.wim or install.esd in the ISO using (if..else) statement.
Let me know if you require any more help!

Reply:
In my opinion, this script is requires one of `install.wim` or `install.esd`, so if you need to check out before copying the image, you can convert the "install.esd" before this message out if "install.esd" was exists and change the message from "install.wim" to "install.wim or install.esd".